ppafm/cpp/Makefile

```diff
@@ -1,9 +1,14 @@
 CC=g++
 LD=g++
+
 #CPPFLAGS= -fPIC -std=c++11 -O2 -mtune=native
 CPPFLAGS= -fPIC -std=c++11 -O2 -mtune=native -fopenmp
+#CPPFLAGS= -fPIC -std=c++11 -Og -g
```

Comment:
Beware, I think that the changes in the makefile are not neccessary.

Reply:
Actually I don't understand why you removed the debug options from the Makefile. I would keep them there, so that we can debug it easily in future.